assignment11/task-manager.js

```diff
@@ -0,0 +1,61 @@
+Task = require('./task');
+
+//proto object that manager will point to - has all the functions (to reserve memory)
+var proto = {};
+//create is a non constructive function that creates a manager object
+var create = function () {
+    //create manager and its prototype will be proto
+    var manager = Object.create(proto);
+    //array to hold tasks initiated by manager
+    manager.tasks = [];
+    //array that contains only active tasks
+    manager.activeTasks = [];
```

Comment:
I just noticed that you add to the active tasks only when the user creates a new task. This will not work correctly if the user edited a task or updated its remaining time.